pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -294,6 +295,22 @@ sleep 5s && ifconfig usb0 down
 
 After this you should be able to easily use your Intel Edison in USB host mode.
 
+## Jetson Devices
+### Custom device trees
+Loading of custom device-trees in balenaOS is supported by the Jetson Nano and Jetson TX2 family of devices. The list of devices that suport this function includes:
+- Jetson Nano SD-CARD
+- Jetson Nano eMMC
+- Jetson Nano 2GB Devkit SD
+- Jetson TX2
+- Jetson TX2 NX (with Xavier NX Devkit)
+
+Loading of custom device-trees is not supported for the Jetson Xavier family of devices in balenaOS. The U-Boot bootloader provides the complete set of functionality necessary for loading custom device-trees in balenaOS for the Jetson Nano and TX2 devices, and this bootloader is not supported by the Jetson Xavier BSP.
+
+To test a custom device-tree on a Jetson Nano or TX2 device, place your custom compiled device tree in the host operating system of your device, in the following path: /mnt/sysroot/active/current/boot/
```

Comment:
Switching to device tree all over the place as you suggested, but not to device tree overlays because this is not supported (unlike on the RaspberryPIs which only support overlays and the dtb is picked by the boot firmware).